src/content/docs/en/guides/integrations-guide/sitemap.mdx

```diff
@@ -119,6 +119,28 @@ Allow: /
 Sitemap: https://<YOUR SITE>/sitemap-index.xml
 ```
 
+If you'd like to generate `robots.txt` dynamically, add a file named `robots.txt.ts`. In the following code, we're using the [`dedent`](https://www.npmjs.com/package/dedent) package for formatting multi-line template literals.
```

Comment:
We do not recommend using the first person in documents.

You can find writing and style Guide in [Astro Docs Docs](https://contribute.docs.astro.build/guides/writing-style/#writing-style)